_docker/docker-compose.yml.erb

```diff
@@ -154,7 +154,23 @@ services:
       - DB_PASSWORD=searchisko
     links:
      - mysql
-  
+
+  #
+  # Environment actions
+  #
+  backup:
+    build: ./awestruct
```

Comment:
Any particular reason to use the awestruct container vs the base ruby container for stuff like this?